src/System.Security.Cryptography.X509Certificates/src/Internal/Cryptography/Pal.Windows/StorePal.Import.cs

```diff
@@ -81,14 +81,14 @@ private static StorePal FromBlobOrFile(byte[] rawData, string fileName, SafePass
                             {
                                 //
                                 // If the user did not want us to persist private keys, then we should loop through all
-                                // the certificates in the collection and set our custom CERT_DELETE_KEYSET_PROP_ID property
+                                // the certificates in the collection and set our custom CERT_CLR_DELETE_KEY_PROP_ID property
```

Comment:
Nit: it's no longer "custom", right?

Reply:
Depends on what you mean.  The property means nothing to Windows, or any device drivers.  They've given us a number that we get to do with what we want.

So it's "not something we expect to be on a certificate, and we fully own the semantics".  Sort of means custom to me.  (But we're no longer squatting on an ID that actually got claimed in Win8, but we happened to have accidentally non-conflicting uses of it)

Reply:
I'll leave this as is, I don't think it's worth CI's time considering ambiguity related to the meaning of the word "custom" - please IM me if you feel strongly I should fix it